packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -249,28 +249,31 @@ export class BreadcrumbBase extends React.Component<IBreadcrumbProps, any> {
   };
 
   private _onRenderItem = (item: IBreadcrumbItem) => {
-    if (item.onClick || item.href) {
+    const { as, href, onClick, isCurrentItem, role, text, ...additionalProps } = item;
+
+    if (onClick || href) {
       return (
         <Link
-          as={item.as}
+          {...additionalProps}
+          as={as}
           className={this._classNames.itemLink}
-          href={item.href}
-          aria-current={item.isCurrentItem ? 'page' : undefined}
+          href={href}
+          aria-current={isCurrentItem ? 'page' : undefined}
           // eslint-disable-next-line react/jsx-no-bind
           onClick={this._onBreadcrumbClicked.bind(this, item)}
-          role={item.role}
+          role={role}
```

Comment:
nit: can remove, `... additionalProps` will cover it

Reply:
Should it be removed from the interface also as the extension will make it redundant?

Reply:
You should leave `role` in the interface. It's an important enough prop to be worth explicitly calling out.

Reply:
> Should it be removed from the interface also as the extension will make it redundant?
I meant that this is redundant. But @ecraig12345's comment is fair, I am okay to leave it as is